binding/python/multiverso/api.py

```diff
@@ -45,6 +45,9 @@ def get(self, size):
     def add(self, data):
         raise NotImplementedError("You must implement the add method.")
 
```

Comment:
Yes, it conflicts with PEP8
I'm wondering why my pep8 checker didn't report it... You really have beaten it... Lol